cli/src/cmd_auth.rs

```diff
@@ -353,46 +376,42 @@ pub struct CmdAuthLogout {
 }
 
 impl CmdAuthLogout {
-    pub async fn run(&self, ctx: &Context) -> Result<()> {
-        if !self.force {
-            match dialoguer::Confirm::new()
-                .with_prompt("Confirm authentication information deletion:")
-                .interact()
-            {
-                Ok(true) => {}
-                Ok(false) => {
-                    println!("Action aborted. No changes have been made.");
-                    return Ok(());
-                }
-                Err(err) => {
-                    return Err(anyhow!("prompt failed: {}", err));
-                }
-            }
+    pub async fn logout(&self, ctx: &Context) -> Result<()> {
+        if !self.force && !yes("Confirm authentication information deletion:")? {
```

Comment:
It'd be nice to confirm that the user is deleting the profile they intend to delete. How about

```console
"Confirm authentication information deletion for profile {PROFILE NAME}:"
```